config/environment.go

```diff
@@ -0,0 +1,10 @@
+package config
+
+const (
+	// Prefix of all environment configurations
+	EnvConfigPrefix = "BLOCKY_"
+	// Environment variable with the path of the config file or folder
+	ConfigFilePath = "BLOCKY_CONFIG_FILE"
+	// Legacy environment variable with the path of the config file or folder
+	ConfigFilePathOld = "CONFIG_FILE"
```

Comment:
I think these make it sound like they're hardcoded paths. Maybe add a `EnvVar` prefix or suffix to the names?

Maybe also take the oppurtinity to remove "file" from the name (`FilePath` -> `Path`) as the env var values can be a directory.